react/src/components/organisms/Header/index.js

```diff
@@ -15,15 +15,33 @@ class Header extends Component {
       utilNavOpen: false
     };
   }
-  menuButtonClicked() {
-    const bodyClass = document.querySelector('body').classList;
-    bodyClass.toggle('show-menu');
-    if (bodyClass.value.length > 0) {
+  menuButtonClicked = () => {
+    // eslint-disable-next-line no-undef
+    const body = document.querySelector('body');
+    let bodyClass;
+    if (body.hasAttribute('class')) {
+      bodyClass = body.getAttribute('class');
+      if (bodyClass) {
+        if (bodyClass.indexOf('show-menu')) {
+          bodyClass = bodyClass.split(' ').filter((val) => val !== 'show-menu').join(' ');
+        } else {
+          bodyClass += ' show-menu';
+        }
+        body.setAttribute('class', bodyClass);
+      } else {
+        bodyClass = 'show-menu';
+        body.setAttribute('class', 'show-menu');
+      }
+    } else {
+      bodyClass = 'show-menu';
+      body.setAttribute('class', 'show-menu');
+    }
+    if (bodyClass) {
```

Comment:
i know this logic was here before--> shouldn't if be checking for mobile something here not that the body of the page has a class attribute?